home-open-api/pom.xml

```diff
@@ -63,13 +63,11 @@
                     <validateOperationDescription>true</validateOperationDescription>
                     <validateOperationTag>true</validateOperationTag>
                     <validateOperationSummary>true</validateOperationSummary>
-                    <validateModelPropertiesExample>true</validateModelPropertiesExample>
+                    <validateModelPropertiesExample>false</validateModelPropertiesExample>
                     <validateModelNoLocalDef>true</validateModelNoLocalDef>
-                    <validateNaming>true</validateNaming>
+                    <validateNaming>false</validateNaming>
```

Comment:
Why did we disable it? I think we must keep it enabled.

Reply:
Changed to "true". Added path, parameter, header and property naming checks as well.